docs/Reference.mdx

```diff
@@ -293,11 +293,13 @@ Timestamps are expressed in seconds and display the given timestamp in the user'
 
 Guild navigation types link to the corresponding resource in the current server.
 
-| Type      | Description                                                                                           |
-|-----------|-------------------------------------------------------------------------------------------------------|
-| customize | _Customize_ tab with the server's [onboarding prompts](#DOCS_RESOURCES_GUILD/guild-onboarding-object) |
-| browse    | _Browse Channels_ tab                                                                                 |
-| guide     | [Server Guide](https://support.discord.com/hc/en-us/articles/13497665141655)                          |
+| Type            | Description                                                                                             |
+|-----------------|---------------------------------------------------------------------------------------------------------|
+| customize       | _Customize_ tab with the server's [onboarding prompts](#DOCS_RESOURCES_GUILD/guild-onboarding-object)   |
+| browse          | _Browse Channels_ tab                                                                                   |
+| guide           | [Server Guide](https://support.discord.com/hc/en-us/articles/13497665141655)                            |
+| linked-roles    | [Linked Roles](https://support.discord.com/hc/en-us/articles/10388356626711) tab                        |
+| linked-roles:ID | _Linked Role_ connection
```

Comment:
```suggestion
| linked-roles:ID | _Linked Role_ connection
```
_Maybe_ this would be better off in the normal reference section but that's just an idea...